src/software/world/robot_state.h

```diff
@@ -77,3 +82,12 @@ class RobotState
     Angle orientation_;
     AngularVelocity angular_velocity_;
 };
+
+/**
+ * A light structure for a robot state with an ID
+ */
+struct RobotStateWithId
+{
+    unsigned int id;
```

Comment:
We can move the `RobotId` typedef from `Robot` to robot_state.h and use that throughout `PhysicsWorld`